packages/eslint-plugin/src/rules/no-redundant-type-constituents.ts

```diff
@@ -102,6 +102,10 @@ function describeLiteralType(type: ts.Type): string {
     return type.value.toString();
   }
 
+  if (tsutils.isIntrinsicErrorType(type) && type.aliasSymbol) {
+    return type.aliasSymbol.escapedName.toString();
```

Comment:
[Bug] This doesn't mention the type as being an "error typed value" the way the other rules do. The message you'd get for e.g. the `NotKnown | 0` test case is:

> ```plaintext
> 'NotKnown' overrides all other types in this union type.
> ```

The message should at least mention it being an error type. Maybe...

> ```plaintext
> 'NotKnown' is an `error` type, which acts as an `any` and overrides all other types in this union type.
> ```

...that's a little wordy, so if you can think of a smoother message, that'd be great. 😄

Reply:
Thank you for the review! I've updated the error message in df04797. Please check it.